clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -133,10 +147,12 @@ class AzureResourceManagerClient extends AzureBaseClient {
     ResourceManagementService.create(this.buildConfiguration(creds))
   }
 
-  private static void createResourceGroupVNet(AzureCredentials creds, String resourceGroupName, String region) {
-    def vNetName = AzureUtilities.VNET_NAME_PREFIX + resourceGroupName
+  private static void initializeResourceGroupVNet(AzureCredentials creds, String resourceGroupName, String region) {
```

Comment:
Is there some duplication between this method and the initializeResourceGroupAndVnet method on line 90?